button/button.css

```diff
@@ -0,0 +1,91 @@
+@import url(../style.css);
+
+.btn {
+    border: none;
+    padding: .5rem 1rem;
+    color: white;
+    font-size: 16px;
+    border-radius: 2px;
+    background-color: #fff;
+}
+
+.btn:hover {
+    cursor: pointer;
+}
+
+.btn-primary {
+    background-color: var(--primary);
+}
+
+.btn-secondary {
+    background-color: var(--secondary);
+}
+
+.btn-success {
+    background-color: var(--success);
+}
+
+.btn-warning {
+    background-color: var(--warning);
+}
+.btn-danger {
+    background-color: var(--danger);
+}
+
+.btn-outline-primary {
+    border: 2px solid var(--primary);
+    color: var(--primary);
+}
+
+.btn-outline-secondary {
+    border: 2px solid var(--secondary);
+    color: var(--secondary);
+}
+
+.btn-outline-success {
+    border: 2px solid var(--success);
+    color: var(--success)
+}
+
+.btn-outline-warning {
+    border: 2px solid var(--warning);
+    color: var(--warning);
+}
+.btn-outline-danger {
+    border: 2px solid var(--danger);
+    color: var(--danger);
+}
+
+.btn-hover.btn-outline-primary:hover {
+    background-color: var(--primary);
+    color: white;
+}
+
+.btn-hover.btn-outline-secondary:hover {
+    background-color: var(--secondary);
+    color: white;
+}
+.btn-hover.btn-outline-success:hover {
+    background-color: var(--success);
+    color: white;
```

Comment:
Instead of writing 'color:white' in individual classes, you can write in one line by calling all classnames separated with commas

Reply:
or you can add all the colors in main class(root)

Reply:
thanks, I'll do that

Reply:
> Instead of writing 'color:white' in individual classes, you can write in one line by calling all classnames separated with commas

DONE